app/gig.py

```diff
@@ -1,19 +1,37 @@
 import os
 import shutil
 import pkg_resources
+import click
 
 
 from .map import MAP
 
 
 class GIG(object):
+    __dest = os.getcwd() + '/.gitignore'
+
     def __init__(self):
         pass
-    
-    def generate(self, arg):
+
+    def gi_not_exists(self,gi_dest):
```

Comment:
space between parametes

Reply:
is `gi_dest` the path of the directory that contains the gitignore or the path of  the gitignore itself?

Reply:
```gi_dest``` is the path of the ```.gitignore``` file